### ISSUE TITLE
Front-end highlighter redesign: Missing screen reader announcements for disable/reenable styles

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1627]
When the styles are disabled or reenabled, focus is shifted back to the menu trigger button and there is no screen reader announcement that the action was successful. This make be connected to https://github.com/equalizedigital/accessibility-checker/issues/1620.

## How to fix
Add a hidden screen-reader only div with role="status" aria-live="polite" aria-atomic="true" and insert messages into it when the styles are disabled or re-enabled.
